provider/resources.go

```diff
@@ -209,3 +211,35 @@ func Provider() tfbridge.ProviderInfo {
 
 	return prov
 }
+
+func editRules(defaults []tfbridge.DocsEdit) []tfbridge.DocsEdit {
+	return append(
+		defaults,
+		rewriteTerraformToPulumi,
+		rewritePermissions,
+	)
+}
+
+// This particular provider elides a lot of descriptions without this rule.
+// TODO: When https://github.com/pulumi/pulumi-terraform-bridge/issues/2251 is implemented, we may remove this rule.
+var rewriteTerraformToPulumi = tfbridge.DocsEdit{
+	Path: "*",
+	Edit: func(_ string, content []byte) ([]byte, error) {
+		return bytes.ReplaceAll(content, []byte("Terraform"), []byte("Pulumi")), nil
```

Comment:
I'm wondering if it would better to do a case insensitive search instead?

Reply:
We don't want to be case insensitive, but we could do 2 passes.
```suggestion
		content= bytes.ReplaceAll(content, []byte("Terraform"), []byte("Pulumi"))
		return bytes.ReplaceAll(content, []byte("terraform"), []byte("pulumi")), nil
```

Otherwise we risk recommending that people run `Pulumi import` or have sentinces like "pulumi supports ..." (From "Terraform supports ...").

Reply:
Unfortunately, I discovered we can't do a lowercase pass at this time, because the import translation logic currently relies on finding `terraform import` 💀. I don't want to get too precious here so I'm going to note this hurdle in the linked bridge issue and merge as-is, since it's an overall improvement here.